src/assets/styles/global/_footer.css

```diff
@@ -0,0 +1,62 @@
+footer {
+    --flow-space: 2.25rem;
+
+    background-color: var(--fl-bgColor, var(--color-indigo-800));
+    color: var(--fl-fgColor, var(--color-white));
+    padding-inline: var(--common-inline-padding);
+    padding-block: var(--common-block-padding);
+}
+
+footer .content {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+
+footer .content > * {
+    width: max-content;
+}
+
+footer section {
+    border-block-start: 0.125rem solid currentColor;
+    padding-block-end: 0;
+    padding-block-start: 1em;
+    padding-inline: 0;
+    border-block-end: 0;
+    box-shadow: none;
+}
+
+footer ul {
+    display: flex;
+    flex-direction: column;
+    gap: 1em;
+}
+
+footer section a {
+    display: inline-flex;
+}
+
+footer svg {
+    fill: var(--fl-linkFgColor, white);
```

Comment:
We need a new rule for footer svg on hover; in UIO contrast themes, the svgs disappear on hover because the fill is the same colour as their background colour. 

We should add:

```
footer a:hover svg {
    fill: var(--fl-linkBgColor, currentColor);
}
```